python/ray/tests/test_resource_demand_scheduler.py

```diff
@@ -3555,10 +3559,11 @@ def test_info_string_with_launch_failures():
  2.00GiB/8.00GiB memory
  3.14GiB/16.00GiB object_store_memory
 
+Constraints:
```

Comment:
Could you add a test where we have multiple bundle shapes so this contains multiple lines?

Reply:
Sure. A new test containing multi-lines of constraints is added in the latest commit.